src/components/Checkbox/Checkbox.vue

```diff
@@ -1,13 +1,52 @@
 <template lang="pug">
+extends ../Base/InputBase.pug
```

Comment:
Can we rename the component to `Choices` rather than `Checkbox`?
Because when we use Checkbox, I want to use it as a single Checkbox (might be conflict with toggle) and it should have True value and False value like here: https://vuejs.org/guide/essentials/forms.html#checkbox-1